### DISCUSSION TITLE
Kafka Connect Sidecar

### DISCUSSION THREAD

**Discussion by Yedidyah27** [strimzi/strimzi-kafka-operator#4933] [Ideas]
Hey,
I would like to run a second container in the same pod as kafka connect. Based on the current implementation this is not possible, am I correct? Is there any plans to add such a feature?

**Comment by scholzj** [comment]
You are right, it is not possible right now. There is a #2976 covering it in general. But there is currently no plan to add such feature.

What is your use case for it?

**Comment by Yedidyah27** [comment]
I have a custom kafka sink connector that needs sensative data that is saved on an external system. My connector makes a service call to it and enriches the message it recieved. I wanted to implement some kind of health check to this external system and updates the connector of its status. I will come up with another solution.
